### PR TITLE
Fixed price field

### PR DESCRIPTION
**Pull request by tavisca-acharpe**

### CODE REVIEW

**Review by tavisca-cpadole — APPROVED** [review]
Looks Good

**Review by tavisca-mrathore — APPROVED** [review]
looks good.

File: src/OnlineRetailPortal.Web/Validations/AddProductRequestValidator.cs [hunk 1]
```diff
@@ -43,7 +43,7 @@ public AddProductRequestValidator()
             .WithErrorCode(ErrorCode.MissingField())
             .WithMessage(Error.MissingField("Category"));
 
-            RuleFor(x => x.Price.Money.Amount)
+            RuleFor(x => x.Price.Amount)
```
Comment (Tavisca-smohite) [inline src/OnlineRetailPortal.Web/Validations/AddProductRequestValidator.cs:46]: should have non-negative check

File: src/OnlineRetailPortal.Web/Validations/UpdateProductRequestValidator.cs [hunk 2]
```diff
@@ -50,7 +50,7 @@ public UpdateProductRequestValidator()
             .WithErrorCode(ErrorCode.MissingField())
             .WithMessage(Error.MissingField("Category"));
 
-            RuleFor(x => x.Price.Money.Amount)
+            RuleFor(x => x.Price.Amount)
```
Comment (Tavisca-smohite) [inline src/OnlineRetailPortal.Web/Validations/UpdateProductRequestValidator.cs:53]: same as above

**Review by Tavisca-smohite — CHANGES_REQUESTED** [review]

File: src/OnlineRetailPortal.Web/Validations/AddProductRequestValidator.cs (hunk 1 above)
Comment (tavisca-acharpe) [reply src/OnlineRetailPortal.Web/Validations/AddProductRequestValidator.cs:46]: done

File: src/OnlineRetailPortal.Web/Validations/UpdateProductRequestValidator.cs (hunk 2 above)
Comment (tavisca-acharpe) [reply src/OnlineRetailPortal.Web/Validations/UpdateProductRequestValidator.cs:53]: done

**Review by Tavisca-smohite — APPROVED** [review]

**Pull request merged**